tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/GraphQLTests.cs

```diff
@@ -164,102 +153,68 @@ protected async Task RunSubmitsTraces(string packageVersion = "")
                     }
                 }
 
-                var spans = graphQLValidateSpans.Concat(graphQLExecuteSpans).ToList();
-                SpanTestHelpers.AssertExpectationsMet(_expectations, spans);
+                var spans = agent.WaitForSpans(expectedSpans);
```

Comment:
Maybe we can order the returned spans by StartTime to improve snapshot stability

Reply:
> Maybe we can order the returned spans by StartTime to improve snapshot stability

`VerifyHelper.VerifySpans()` does that 🙂